content/en/contribute/medic/testing/quality-assistance.md

```diff
@@ -0,0 +1,55 @@
+---
+title: "Quality Assistance"
+linkTitle: "Quality Assistance"
+weight: 1
+description: >
+  How the Quality Assistance process works at Medic
+---
```

Comment:
It's not clear to me why this page is in a separate section... can we move it to within _Product Development Process_?

Reply:
Yes, it makes sense to move it. It was created before the product development migration from Notion, hence the confusion.